how-to/how-to-install.md

```
@@ -22,15 +22,17 @@ desired folder name and the `--locale=it_IT` with your desired locale. You can o
 
 In this step, we will generate a config file and set up the database
 credentials for our installation.
-The basic syntax of the command is the following: `wp config create --dbname=<dbname> --dbuser=<dbuser> [--dbpass=<dbpass>]`
+The basic syntax of the command is the following: `wp config create --dbname=<dbname> --dbuser=<dbuser> [--dbhost=<dbhost>] [--dbpass=<dbpass>]`
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)